src/app/SIPDNS/SIPDNSManager.cs

```diff
@@ -297,10 +401,11 @@ public static async Task<SIPDNSLookupResult> ResolveAsync(SIPURI sipURI)
                                 //}
                                 //else
                                 //{
-                                SIPDNSServiceResult nextSRVRecord = sipLookupResult.GetNextUnusedSRV();
-                                int lookupPort = (nextSRVRecord != null) ? nextSRVRecord.Port : port;
-                                return DNSARecordLookup(nextSRVRecord, host, lookupPort, false, sipLookupResult.URI);
+                                //   SIPDNSServiceResult nextSRVRecord = sipLookupResult.GetNextUnusedSRV();
+                                //   int lookupPort = (nextSRVRecord != null) ? nextSRVRecord.Port : port;
+                                //   return DNSARecordLookup(nextSRVRecord, host, lookupPort, false, sipLookupResult.URI);

```

Comment:
There is a lot of commented out code in the PR can it be deleted?

Reply:
originial code is kept there for review/reference of my changes
original author may delete obsolete/stale code/comments